libs/common/src/lib/models.ts

```diff
@@ -227,7 +227,7 @@ export function validateXContestAccount(id: string): string | false {
 }
 
 // Validates a Meshtastic ID.
-export function validateMeshBirAccount(id: string): string | false {
+export function validateBircomAccount(id: string): string | false {
```

Comment:
I think you can revert this change
- You would need to update all the call places
- All the code use MeshBir